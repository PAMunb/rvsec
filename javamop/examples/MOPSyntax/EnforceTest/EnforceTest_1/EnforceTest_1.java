public class EnforceTest_1 implements Runnable {
    public void run(){
        System.out.println("Thread " + Thread.currentThread().getName());
    }

    public static void main(String args[]) throws Exception{

        Thread t1 = new Thread (new EnforceTest_1());
        Thread t2 = new Thread (new EnforceTest_1());
        t1.setName("T1");
        t2.setName("T2");

        t1.start();
        t2.start();

        t1.join();
        t2.join();
    }
}
