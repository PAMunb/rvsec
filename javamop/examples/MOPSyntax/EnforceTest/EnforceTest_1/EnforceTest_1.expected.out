Thread T2
2 end
2 end
Thread T1
1 end
1 end
